storagenode/storagenodedb/piecespaceused.go

```diff
@@ -0,0 +1,161 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedb
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storagenode/pieces"
+)
+
+type pieceSpaceUsedDB struct {
+	*InfoDB
+}
+
+// PieceSpaceUsedDB returns database for storing piece expiration data
+func (db *DB) PieceSpaceUsedDB() pieces.PieceSpaceUsedDB { return db.info.PieceSpaceUsedDB() }
+
+// PieceSpaceUsedDB returns database for storing piece expiration data
+func (db *InfoDB) PieceSpaceUsedDB() pieces.PieceSpaceUsedDB { return &db.pieceSpaceUsedDB }
```

Comment:
why wrap twice?

Reply:
i updated the comments for clarity. One is exposing the DB from infoDb, and the other is exposing via the storagenode master db